api/bin/chainflip-ingress-egress-tracker/src/witnessing/state_chain.rs

```diff
@@ -84,6 +104,22 @@ enum WitnessInformation {
 		tx_out_id: TransactionId,
 		tx_ref: TransactionRef,
 	},
+	VaultDeposit {
```

Comment:
think it makes sense to call this `Swap` as it has an output asset
```suggestion
	VaultSwap {
```

Reply:
I like `VaultDeposit` better because it is more consistent with `VaultDepositWitness`.

Reply:
Don't have a lot of context but, If we have a vault deposit for an LP it would be called a VaultDeposit? which would make more sense, and then this being VaultSwap makes sense

Reply:
i dont have a strong opinion on this - `VaultDeposit` is fine for me too if you prefer it. feel free to resolve